src/modules/Auth/ducks/token.js

```diff
@@ -0,0 +1,21 @@
+import Duck from 'services/Duck';
+import { apiPaths } from 'modules/Auth/const';
+
+const actionCoreName = 'AU_TOKEN';
+
+const token = new Duck(
+  {
+    name: actionCoreName,
+    urlBuilder: apiPaths.refresh,
+  },
+);
+
+const actions = token.actions;
+const reducer = token.reducer;
+
+export default {
+  actions: {
+    refresh: actions.create,
```

Comment:
This is not enough. REST returns updated token, which needs to be placed into LS, etc